watertap/unit_models/zero_order/cooling_tower_zo.py

```diff
@@ -0,0 +1,60 @@
+###############################################################################
+# WaterTAP Copyright (c) 2021, The Regents of the University of California,
```

Comment:
Should the copyright date in the header still be 2021 even though it is now 2022?

Reply:
I didn't pay attention to this before, but all our migrated models from WT3 use the same copyright. @adam-a-a

Reply:
We should not change the copyright header ourselves. Firstly, there is a lot of paperwork that goes into getting it approved, and secondly there is an automated tool for updating the headers when required.